src/id8_i/startup.py

```diff
@@ -112,6 +112,24 @@ def _startup_create_devices_plan():
 
 RE(_startup_create_devices_plan())
 
+pv_registers = oregistry["pv_registers"]
+
+spec_file_name = pv_registers.spec_file
+spec_file_name.wait_for_connection()
+_fname = spec_file_name.get()
+
+if len(_fname) > 0 and _fname.endswith(".dat"):
```

Comment:
```suggestion
if len(_fname) > 4 and _fname.endswith(".dat"):
```

So it has more than just `.dat`